apps/api/src/form/form.controller.ts

```diff
@@ -43,4 +50,39 @@ export class FormController {
   async getFormById(@Param('id') id: string): Promise<FormEntity> {
     return await this.formService.getFormById(id);
   }
+
+  @ApiOperation({
+    summary: 'Export Data',
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @ApiResponse({ status: HttpStatus.CREATED, type: EmptyResponse })
+  @HttpCode(HttpStatus.CREATED)
+  @Get('/export/:passCode')
+  async exportAll(@Param('passCode') passCode: string, @Res() res: Response) {
+    // TODO: Add logging
+
+    // TODO: Based on auth roles in the future.
+    if (passCode === process.env.EXPORT_SECRET) {
```

Comment:
We can save some nesting by throwing early rather than surrounding the whole block in an if statement 

```
    if (passCode !== process.env.EXPORT_SECRET) {
        throw new....
    }
```